vivisect/analysis/elf/elfplt.py

```diff
@@ -59,13 +62,17 @@ def analyze(vw):
     Do simple linear disassembly of the .plt section if present.
     Make functions
     """
+    if vw.getMeta('Platform') == 'qnx':
+        logger.warning('skipping initial PLT section analysis for QNX binary.')
+        return
+
     for sva, ssize in getPLTs(vw):
         analyzePLT(vw, sva, ssize)
 
 
-def getGOT(vw, fileva):
+def getGOT(vw, filename):
```

Comment:
to keep consistent with other places in the codebase, can you make this function `gotGOTByFileName` or something simliar?

It would also avoid confusion between `getGOTs` and `getGOT`

Reply:
done